src/menu_specialized.c

```diff
@@ -1474,9 +1473,82 @@ static void sub_81D32F4(struct Sprite *sprite)
     }
 }
 
-// Todo: Move these variables to C.
-extern const s16 gUnknown_08625B2C[][2];
-extern const struct SpriteTemplate gUnknown_08625B14;
+static const struct OamData sOamData_8625AD0 =
+{
+    .y = 0,
```

Comment:
Use the constants for `affineMode`, `objMode`, `bpp`, and the `SPRITE_SIZE/SHAPE` macros for  `shape`, and `size`.